### ISSUE TITLE
Wrong autocorrection for Style/RedundantInterpolation when interpolation is a method call without parentheses

### ISSUE THREAD

**Issue by existentialmutt** [rubocop/rubocop#10276]
The following line is not correctly handled by the autocorrect behavior for `Style/RedundantInterpolation`

```ruby
"#{my_method arg}"
```

Running `rubocop -a` changes it to
```ruby
my_method arg.to_s
```

The correct replacement is
```ruby
my_method(arg).to_s
```

--------

## Expected behavior

Either the autocorrect behavior for Style/RedundantInterpolation should be updated to handle method calls without parens, or the rule should be marked SafeAutocorrect: false

## Actual behavior

Running `rubocop -a` on a file containing a redundant interpolation can result in an incorrect replacement as shown above.

## Steps to reproduce the problem

1.  Save a new file `redundant_interpolation_test.rb` containing the following:
```ruby
"#{my_method arg}"
```

2.  run `rubocop -a redundant_interpolation_test.rb`

3.  Observe that the file has been changed to

```ruby
my_method arg.to_s
```

## RuboCop version

```
bundle exec rubocop -V
1.22.3 (using Parser 3.0.3.1, rubocop-ast 1.13.0, running on ruby 2.6.6 x86_64-linux)
  - rubocop-performance 1.11.5
```

**Comment by existentialmutt** [comment]
That was fast!  Thanks @koic !
